Eluant/LuaRuntime.cs

```diff
@@ -570,7 +594,15 @@ private LuaVararg Call(IList<LuaValue> args)
 
                 needEnterClr = true;
                 OnEnterLua();
-                if (LuaApi.lua_pcall(LuaState, args.Count, LuaApi.LUA_MULTRET, 0) != 0) {
+
+                // Include a lua stacktrace if an error is encountered
+                LuaApi.lua_getglobal(LuaState, "debug");
```

Comment:
Sorry I missed this the first time around. :(

If the code is sandboxed then the debug global might be erased, or untrusted code might replace `debug.traceback` with a malicious version.  It would be wise instead to store a reference to the `debug.traceback` Lua function in the registry when the `LuaRuntime` object is first initialized to solve both problems.